scripts/split_and_preprocess.py

```diff
@@ -0,0 +1,51 @@
+# split_and_preprocess.py
```

Comment:
Looks like we don't have a specific step of pre-processing as Tiff did as she made a separate preprocessing with StandardScaler and saved the scaled data. 

Should we consider doing the same? If not, maybe change the name to avoid confusion?